scvi/inference/fish.py

```diff
@@ -13,17 +13,15 @@
 class FishPosterior(Posterior):
 
     @torch.no_grad()
-    def elbo(self, verbose=False):
+    def elbo(self):
         elbo = compute_elbo(self.model, self, mode="smFISH")
-        if verbose:
-            logging.info("ELBO Fish: %.4f" % elbo)
+        logging.debug("ELBO Fish: %.4f" % elbo)
```

Comment:
As discussed, we should use a per-module logger ``logger = logging.getLogger(__name__)``
 declared right after the imorts and then use calls such ``logger.info(..)``

Reply:
Oh so no logger.debug ?

Reply:
yes yes, all logs should be made using the module logger, ie ``logger.debug`` , ``logger.warning``